File: statistics/src/main/go/com.github/dto/login_dto.go
package dto

// LoginDto swagger:model
type LoginDto struct {
	Id        uint      `json:"id" example:"1"`
	UserId    int64     `json:"userId,omitempty" example:"12"`
	Device    string    `json:"device" example:"phone"`
	Location  string    `json:"location" example:"192.168.1.1"`
}

File: admins/src/main/java/com/github/admins/dto/LoginDto.java
package com.github.admins.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.swagger.annotations.ApiModelProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LoginDto implements Serializable, Cloneable {

    private static final long serialVersionUID = 5407992044101536119L;

    @ApiModelProperty(
            value = "Login id",
            example = "1"
    )
    @JsonProperty(value = "id")
    private Long id;

    @JsonProperty(value = "createAt")
    private Date createAt;

    @JsonProperty(value = "device")
    private String device;

    @JsonProperty(value = "location")
    private String location;

}
